stock_request_picking_type/models/stock_request_order.py

```diff
@@ -7,43 +7,32 @@
 class StockRequestOrder(models.Model):
     _inherit = "stock.request.order"
 
-    @api.model
-    def _get_default_picking_type(self):
-        companies = self._context.get("allowed_company_ids", []).copy()
-        companies.append(False)
-        return (
-            self.env["stock.picking.type"]
-            .search(
-                [
-                    ("code", "=", "stock_request_order"),
-                    "|",
-                    ("warehouse_id.company_id", "in", companies),
-                    ("warehouse_id", "=", False),
-                ],
-                limit=1,
-            )
-            .id
-        )
-
     picking_type_id = fields.Many2one(
         comodel_name="stock.picking.type",
         string="Operation Type",
-        default=_get_default_picking_type,
+        compute="_compute_picking_type_id",
         required=True,
+        store=True,
```

Comment:
You should add readonly=False too.

Reply:
> You should add readonly=False too.

For the default for computed fields is readonly=True

Reply:
Fix it!